### ISSUE TITLE
Build DockerFile to include migrating our tables and views

### ISSUE THREAD

**Issue by zachmandeville** [apisnoop/snoopDB#2]
Create a simple DockerFile that has all necessary programs at startup to be able to initialize all our tables, functions, and views

We can start with the DockerFile from cncf/apisnoop, but remove any unnecessary dependencies or obfuscations.

We can say this ticket is done when one can start up postgres with
```shell
docker run -p 5432:5432 -e POSTGRES_USER="something" POSTGRES_PASSWORD="somethingelse" snoop
```
and the resulting db has `plpython3u` and `pg-semver` already added as extension and the snoopUtils programs available to the python functions.

the subsequent creation of all sql files will be handled in #1

**Comment by zachmandeville** [comment]
my intention is to setup every sql file and data load to be in the docker-entrypoint-initdb.d folder of the postgres docker container.  This means everythign can get up and running with a simple docker run command.  

Currently, all tables and functions are being loaded this way, along with 3 helper scripts that load up all the data we need.  
Next is to work on the views we need (these need more comments and so are going a bit slower).
Lastly, I want to try to create the json as part of the initialization.  This would be super slick for our automated jobs later.

I am tracking the progress on this in reports/tickets/ (best viewed in emacs)

**Comment by zachmandeville** [comment]
Implemented with #3
